src/main/java/com/passbook/sparkeighteen/config/SwaggerConfig.java

```diff
@@ -0,0 +1,38 @@
+package com.passbook.sparkeighteen.config;
+
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+import springfox.documentation.builders.ApiInfoBuilder;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfig {
+    @Bean
```

Comment:
Why did we use @Bean here any significance?

Reply:
- Spring @Bean Annotation is applied on a method to specify that it returns a bean to be managed by Spring context.
- Bean annotation is usually declared in Configuration classes methods.